support/@trustfractal/data-host/src/popup/services/StorageService/index.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  ERROR_CLEAR,
+  ERROR_GET_ITEM,
+  ERROR_HAS_ITEM,
+  ERROR_REMOVE_ITEM,
+  ERROR_SET_ITEM,
+} from './Errors';
```

Comment:
Let's change how this service works to use subclasses of Error like the other services.